### ISSUE TITLE
extra camera in Janson

### ISSUE THREAD

**Issue by yoe** [FOSDEM/video#105]
It would be nice to have a second camera in Janson.

There are a few issues that were brought up in response to this suggestion during the meeting:

* Legal issues: will people who are recorded in that room not object:
* It makes Janson aan exception regarding the workflow
* It would require extra manpower which we might not have

As regards the last point, I will note that the DebConf video team is planning a Sprint in Diegem the week before the event; some manpower might be gotten from there.

Beyond that, the video team should probably consider this suggestion and decide what to do...

**Comment by krokodilerian** [comment]
Honestly, I've always wanted to try that. If we have the manpower to actually man the room, I'll do the necessary tests to see how to utilize three boxes.

Last time I player with it, I didn't have the right amount of CPU time to process everything, but voctomix was able to handle the three streams.

**Comment by RichiH** [comment]
My suggestion back at the postmortem was to do this with a different stack,
namely Nageru, to get a feeling for that workflow.

Richard

Sent by mobile; excuse my brevity and the wall of text Gmail appends by
default.

**Comment by yoe** [comment]
On Sun, Sep 24, 2017 at 01:16:39PM -0700, Richard Hartmann wrote:
> My suggestion back at the postmortem was to do this with a different stack,
> namely Nageru, to get a feeling for that workflow.

That would make the difference between the different rooms even larger
than if we were to stick to Voctomix. I fail to see the benefit in that.

-- 
Could you people please use IRC like normal people?!?

  -- Amaya Rodrigo Sastre, trying to quiet down the buzz in the DebConf 2008
     Hacklab

**Comment by SuperQ** [comment]
On Mon, Sep 25, 2017 at 2:01 AM, Wouter Verhelst <notifications@github.com>
wrote:

> On Sun, Sep 24, 2017 at 01:16:39PM -0700, Richard Hartmann wrote:
> > My suggestion back at the postmortem was to do this with a different
> stack,
> > namely Nageru, to get a feeling for that workflow.
>
> That would make the difference between the different rooms even larger
> than if we were to stick to Voctomix. I fail to see the benefit in that.


Having an advanced/experimental stack as a canary/test is a good idea.  But
I would normally set it up in parallel to the existing setup.

**Comment by krokodilerian** [comment]
Having Janson as the test for something we haven't tried in production is a bad idea for me. If there's someone to do the parallel setup, I'm ok, otherwise no. The two-camera setup is easy to configure and very easy to fall back from to the normal setup, that's the reason I'm ok with it.

**Comment by RichiH** [comment]
@yoe The plan from the post-mortem was, of course, to do two in parallel. The current system and the testing system, right next to each other.

**Comment by yoe** [comment]
@RichiH we're already having issues finding enough volunteers -- in fact, that was one of the arguments brought up in opposition to this suggestion. In that context, adding another setup in parallel is insane.

Nageru is cool, but it is not a panacea. Vocto works, and we have used it last year. 'Nuf said.

**Comment by markvdb** [comment]
Second camera not going to happen, sorry. We can't properly handle the extra complexity.

**Comment by yoe** [comment]
Any chance to keep this open for next year then? I understand if you say 'never ever ever', but just thought I'd ask first ;-)

**Comment by yoe** [comment]
On Sat, Dec 30, 2017 at 10:36:07AM +0000, Vasil Kolev wrote:
> @yoe, the issue is that i couldn't find a way to synchronize the video streams
> from both cameras, and as the audio comes from one of them, anything that shows
> a person speaking on the other will be desynchronized and weird-looking. I
> think you can see it on this year's 34c3 streams sometimes, the delay is less
> than 200ms, I think, but still pretty noticeable.
> 
> This isn't an issue with the slides box/stream, as there's little sync needed
> with that - there is no sound coming from it most of the time, so you can't see
> the ~200ms difference between them.

Understood. Yes, there were some significant video sync issues last year
indeed; I imagine it would be more of a problem with two cameras. Unless
we fix that problem first, I agree we should not give this priority.

> Basically, syncing two cameras in our case would mean either in-room mixer with
> sdi/hdmi, completely changing the protocol over IP (and using lots more
> bandwidth), or some magic I don't know of (like timestamps in the stream and
> synced clocks).

There are ways of sending timestamps in streams, yes. I don't know the
details on them, however, but it might be something to investigate when
we have more time.

-- 
Could you people please use IRC like normal people?!?

  -- Amaya Rodrigo Sastre, trying to quiet down the buzz in the DebConf 2008
     Hacklab

**Comment by RichiH** [comment]
Important point: we should have alerts to see if all clocks are in sync. If
that's part of the setup now, we can rely on this forevermore.

On mobile; can someone give me an issue for that?
